WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -2,31 +2,105 @@
 <div class="container">
   <h1><%= yield(:title) %></h1>
 
-  <% if @everything_good %>
+  <% if @everything_good && @total_missing_outdated == 0 %>
     <%= alert :success, "We're fine. We're all fine here, now, thank you. How are you?" %>
+  <% elsif @everything_good %>
+    <%= alert :warning, "The server is alright, but some translations would enjoy an update." %>
   <% else %>
     <%= alert :danger, "Check below, something may be wrong." %>
   <% end %>
+<% kind_class = @oldest_job_that_should_have_run_by_now ? "danger" : "success" %>
+    <div class="panel panel-<%= kind_class %>">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <span class="label label-<%= kind_class %>">Jobs</span>
 
-  <ul>
-    <li>
-      <% if @oldest_job_that_should_have_run_by_now %>
-        <span class="label label-warning">Jobs</span>
+          <% if @oldest_job_that_should_have_run_by_now %>
+            Uh oh!
+            Job <%= @oldest_job_that_should_have_run_by_now.id %> was created
+            <%= time_ago_in_words @oldest_job_that_should_have_run_by_now.created_at %>
+            ago and still has not run.
+            <%= @jobs_that_should_have_run_by_now.count %>
+            <%= "job".pluralize(@jobs_that_should_have_run_by_now.count) %>
+            <%= "is".pluralize(@jobs_that_should_have_run_by_now.count) %>
+            waiting to run.
 
-        Uh oh!
-        Job <%= @oldest_job_that_should_have_run_by_now.id %> was created
-        <%= time_ago_in_words @oldest_job_that_should_have_run_by_now.created_at %>
-        ago and still has not run.
-        <%= @jobs_that_should_have_run_by_now.count %>
-        <%= "job".pluralize(@jobs_that_should_have_run_by_now.count) %>
-        <%= "is".pluralize(@jobs_that_should_have_run_by_now.count) %>
-        waiting to run.
-
-      <% else %>
-        <span class="label label-success">Jobs</span>
-
-        Looking good!
-      <% end %>
-    </li>
-  </ul>
+          <% else %>
+            Looking good!
+          <% end %>
+        </h3>
+      </div>
+    </div>
+  <% (I18n.available_locales - [:en]).each do |l| %>
+    <% missing = @status_locales[l][:missing] %>
+    <% outdated = @status_locales[l][:outdated] %>
+    <% all_empty = (missing.empty? && outdated.empty?) %>
+    <% class_panel, class_missing, class_outdated, title = panel_classes(missing.empty?, outdated.empty?) %>
+    <div class="panel panel-<%= class_panel %>">
+      <div class="panel-heading heading-as-link">
+        <h3 class="panel-title">
+        <% unless all_empty %>
+          <a data-toggle="collapse" href="#collapse-<%= l %>">
```

Comment:
@jfly Not sure if it was explicit enough, but I'll wait until you have a look at the "new" way of collapsing the panel before merging this ;)